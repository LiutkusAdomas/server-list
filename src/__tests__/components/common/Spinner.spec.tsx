import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import { Spinner } from '../../../components/common/Spinner/Spinner';

describe('Spinner component', () => {
  it('should render a spinner', () => {
    const { asFragment } = render(<Spinner />);
    expect(asFragment).toMatchSnapshot();
  });
});
