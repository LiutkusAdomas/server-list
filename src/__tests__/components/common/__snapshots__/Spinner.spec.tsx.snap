// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Spinner component > should render a spinner 1`] = `[Function]`;
